### ISSUE TITLE
leptonica 1.75.1 says LIBLEPT_PATCH_VERSION is 0

### ISSUE THREAD

**Issue by ilovezfs** [DanBloomberg/leptonica#299]
```
iMac-TMP:homebrew-core joe$ brew install -i leptonica
==> Downloading https://github.com/DanBloomberg/leptonica/releases/download/1.75.1/leptonica-1.75.1.t
Already downloaded: /Users/joe/Library/Caches/Homebrew/leptonica-1.75.1.tar.gz
==> Entering interactive mode
Type `exit' to return and finalize the installation
Install to this prefix: /usr/local/Cellar/leptonica/1.75.1
bash-3.2$ grep -r LIBLEPT_PATCH_VERSION .
./src/allheaders.h:#define LIBLEPT_PATCH_VERSION   0
./src/allheaders_top.txt:#define LIBLEPT_PATCH_VERSION   1
./src/utils1.c:           LIBLEPT_MAJOR_VERSION, LIBLEPT_MINOR_VERSION, LIBLEPT_PATCH_VERSION,
./src/utils1.c:             LIBLEPT_MINOR_VERSION, LIBLEPT_PATCH_VERSION);
bash-3.2$ 
```
So the Homebrew test block fails
```
==> brew test leptonica --verbose
==> FAILED
Testing leptonica
/usr/bin/sandbox-exec -f /tmp/homebrew20180201-89158-1ln3zdx.sb /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/bin/ruby -W0 -I /usr/local/Homebrew/Library/Homebrew -- /usr/local/Homebrew/Library/Homebrew/test.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/leptonica.rb --verbose
==> /usr/bin/clang++ test.cpp -I/usr/local/Cellar/leptonica/1.75.1/include/leptonica -Os -w -pipe -march=native -mmacosx-version-min=10.13
Error: leptonica: failed
<"1.75.1"> expected but was
<"1.75.0">.

diff:
- 1.75.1
?      ^
+ 1.75.0
?      ^
```
which runs
```
  test do
    (testpath/"test.cpp").write <<-EOS
    #include <iostream>
    #include <leptonica/allheaders.h>

    int main(int argc, char **argv) {
        std::fprintf(stdout, "%d.%d.%d", LIBLEPT_MAJOR_VERSION, LIBLEPT_MINOR_VERSION, LIBLEPT_PATCH_VERSION);
        return 0;
    }
    EOS

    flags = ["-I#{include}/leptonica"] + ENV.cflags.to_s.split
    system ENV.cxx, "test.cpp", *flags
    assert_equal version.to_s, `./a.out`
  end
```
```

**Comment by DanBloomberg** [comment]
Thank you for so quickly pointing this out!!

Replaced the configure-make tarball in release 1.75.1 with a tarball with the correct patch number in allheaders.h.

**Comment by ilovezfs** [comment]
@DanBloomberg you're welcome! Thanks for fixing it so quickly. I have now shipped 1.75.1 in Homebrew.
